src/lib/fonts/fonts-dialog.rb

```diff
@@ -298,8 +300,8 @@ def handle_presets_button(widget, event)
         if CWMTab.CurrentTab == "specimens"
           initialize_specimen_widget("")
         elsif CWMTab.CurrentTab == "algorithms"
-          initialize_aaoff_checkbox("")
-          initialize_aamonooff_checkbox("")
+          initialize_antialias_checkbox("")
+          initialize_antialias_mono_checkbox("")
```

Comment:
maybe use default param with `""` so in this case you call only `initialize_antialias_mono_checkbox`